python/doc/examples/reliability_sensitivity/design_of_experiments/plot_plot_design.py

```diff
@@ -1,112 +0,0 @@
-"""
```

Comment:
I do not agree with this change, that is, removing the `PlotDesign` method. This is because `PlotDesign` shows how a `Sample` fills the space. This is an essential property of some space filling designs, such as LHS and low discrepancy sequences. See [^Santner2003] chapter 5 page 121 for different methods on this topic. Hence, providing this feature in the library is very different from having X/YTicks (which is a good thing). It provides not only a graphical value, but a scientific value for the user. 

[^Santner2003]: Santner, T. J., Williams, B. J., Notz, W. I., & Williams, B. J. (2003). The design and analysis of computer experiments (Vol. 1). New York: Springer.